### PR TITLE
Basic components and Routes

### PR DESCRIPTION
**Pull request by raieegulhane**
Added Navbar and footer components.
Added Routes.

### CODE REVIEW

File: src/routes/Routes.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import Wishlist from "../pages/Wishlist"
+import Login from "../pages/Login"
+import SignUp from "../pages/Signup"
+
+function SiteRoutes () {
+    return (
+        <Routes>
+            <Route path="/mockman" element={<Mockman />} />
+            <Route path="/" element={<Home />} />
+            <Route path="/" element={<Home />} />
```
Comment (muskan-gupta-0803) [inline src/routes/Routes.js:18]: Line 18 is repeated.

**Review by muskan-gupta-0803 — APPROVED** [review]
Just a minor change.
Apart from that well structured files. Good to go for now.

**Comment by rahul31agrawal** [comment]
Looks good to merge

**Comment by ArpitPatni** [comment]
Code is well structured ,color variables has been used throughout the project.Looks good to merge!

**Pull request merged**